pages/profiles/cramartef.md

```diff
@@ -0,0 +1,19 @@
+# Fetra Marc Humbert Rahajason
+
+Madagascar/GMT+3.00, East Africa Time.
+
+## I am an OLE leader and Education consultant.
+
+**Language skills**
+
+* Speak both french and English.
+* Translator/interpreter.
+
+**Programming and algorithms:**
+
+* learnt `MS-DOS, Pascal, C, C++, Matlab`
+* learning `HTML, CSS, latex, java`  
+---
+>I would like to help children especially disadvataged ones through education, give them equal chance to create their own future and improve their own lives.
+
+**E-mail:** [Fetra Marc Humbert Rahajason](humbert.fetra@outlook.com)
```

Comment:
The email link is not working. To mak the email link work as intended, you need to add `mailto:` before the email address:
```
[Fetra Marc Humbert Rahajason](mailto:humbert.fetra@outlook.com)
```
or use [autolink](https://daringfireball.net/projects/markdown/syntax#autolink):
```
<humbert.fetra@outlook.com>
```
Reference:
[Daring Fireball - Markdown: Syntax - Automatic Links](https://daringfireball.net/projects/markdown/syntax#autolink)
[GitHub Flavored Markdown Spec - Autolinks](https://github.github.com/gfm/#autolinks)

Reply:
Thanks @xyb994. I see.